liquidator/liquidator.js

```diff
@@ -26,6 +29,10 @@ class Liquidator {
     // Update the client to get the latest position information.
     await this.empClient._update();
 
+    // Update the gasEstimator to get the latest gas price data.
+    // If the client has a data point in the last 60 seconds returns immediately.
+    await this.gasEstimator._update();
```

Comment:
Can you call async methods in a constructor?

Reply:
this pattern hasn't changed. works fine with the `empClient` and my tests of the `gasEstimator` show that it works

Reply:
cool